script.js

```diff
@@ -1,4 +1,90 @@
-var main = function (input) {
-  var myOutputValue = 'hello world';
+var GAME_STATE_DICE_ROLL = 'GAME_STATE_DICE_ROLL';
+var GAME_STATE_CHOOSE_DICE_ORDER = 'GAME_STATE_CHOOSE_DICE_ORDER';
+var GAME_STATE_COMPARE_SCORE = 'GAME_STATE_COMPARE_SCORE';
+var gameState = GAME_STATE_DICE_ROLL;
+
+var currentPlayerRolls = [];
+var currentPlayer = 1;
+var allPlayersScore = [];
+
+var rollDice = function () {;
+  var randomDecimal = Math.random() * 6;
+  var resultInteger = Math.floor(randomDecimal);
+  var diceNumber = resultInteger + 1;
+  console.log ('rollDice', diceNumber);
+  return diceNumber;
+}
+
+var rollDiceForPlayer = function (){;
+  var counter = 0;
+  while (counter < 2){
+    currentPlayerRolls.push(rollDice());
+    counter = counter + 1;
+    gameState = GAME_STATE_CHOOSE_DICE_ORDER;
+  };
+  return `Welcome to Beat That Player ${currentPlayer}!<br><br>Your Dice 1 rolled: ${currentPlayerRolls[0]} & Dice 2 rolled: ${currentPlayerRolls[1]} <br><br>Now input either '1' or '2' to choose the dice number you want as the first digit of your final value.`;
+}
+
+var getPlayerScore = function (input){
+  var currentPlayerScore;
+  if (input == '1'){;
+    var playerScore = Number(String(currentPlayerRolls[0]) + String(currentPlayerRolls[1]));
+    currentPlayerScore = `Your chosen value is ${playerScore}`
+  }
+  if (input == '2'){;
+    var playerScore = Number(String(currentPlayerRolls[1]) + String(currentPlayerRolls[0]));
+    currentPlayerScore = `Your chosen value is ${playerScore}`
+  }
+  allPlayersScore.push(currentPlayerScore);
```

Comment:
It's great that you're saving each players' score into an array! It's a neat idea to do it! in which index 0 is first player, and index 1 is second player